testcafe/tests/helpers/utils.js

```diff
@@ -1,7 +1,25 @@
 import { ClientFunction, Selector, t } from 'testcafe'
 
-export const TESTCAFE_PHOTOS_URL = process.env.TESTCAFE_PHOTOS_URL
-export const TESTCAFE_DRIVE_URL = process.env.TESTCAFE_DRIVE_URL
+const INSTANCE_TESTCAFE = process.env.INSTANCE_TESTCAFE
+export let TESTCAFE_PHOTOS_URL = ''
+export let TESTCAFE_DRIVE_URL = ''
+
+if (INSTANCE_TESTCAFE.includes('tools')) {
+  //Local server
+  TESTCAFE_PHOTOS_URL = `http://photos.${process.env.INSTANCE_TESTCAFE}`
+  TESTCAFE_DRIVE_URL = `http://drive.${process.env.INSTANCE_TESTCAFE}`
+} else {
+  //not local server!
+  TESTCAFE_PHOTOS_URL = `https://${process.env.INSTANCE_TESTCAFE.slice(
+    0,
+    9
```

Comment:
I'm not super comfortable with slicing strings using magic number like that. It's not very easy to read and If we rename the test instances this will break.
Something [like this](https://repl.it/@y_lohse/DistinctRecentBinary) would be better I think.